packages/design-system/src/components/Review/Review.stories.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+
+import Review from './Review';
+
+export default {
+  title: 'Patterns/Review',
+  component: Review,
+  argTypes: {
+    headingLevel: {
+      control: {
+        type: 'select',
+      },
+      options: [1, 2, 3, 4, 5, 6],
```

Comment:
I'm seeing a console error about invalid prop type. I think the component is expecting a string but is getting a number for the `headingLevel` value